### PR TITLE
410 -  Fix Sass linting errors

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Fix Sass linting errors which were breaking the build.

## QA

Pull down code and run `gulp test` from project root. There should be no errors.

## Details

Fixes: #410

### CODE REVIEW

File: docs/partials/footer.hbt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <footer class="pl__footer" role="contentinfo">
     &copy; 2016 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.
     <ul class="inline-list">
-        <li><a href="http://www.ubuntu.com/legal">Legal Information</a></li>
-        <li><a href="https://github.com/ubuntudesign/vanilla-framework/issues/new">Report a bug on this site</a></li>
+        <li class="inline-item"><a href="http://www.ubuntu.com/legal">Legal Information</a></li>
```
Comment (anthonydillon) [inline docs/partials/footer.hbt]: Is this component still to be BEMed? It should be `inline-list__item`

**Comment by anthonydillon** [comment]
Looks good @barrymcgee. One small comment and tests are failing.

**Comment by barrymcgee** [comment]
@anthonydillon Good spot! 👀

**Comment by anthonydillon** [comment]
Looks good thanks :+1:

**Pull request merged**